README.md

```diff
@@ -26,11 +25,11 @@ int main()
 
     TabSample tab;
     // explicit all_of(tab)
-    for(const auto& row : db.run(select(all_of(tab)).from(tab).where(true)))
+    for(const auto& row : db(select(all_of(tab)).from(tab).where(true)))
     {
         int64_t alpha = row.alpha;
-        bool beta = row.beta;
-        std::string gamma = row.gamma;
+        std::string beta = row.beta;
+        bool gamma = row.gamma;
```

Comment:
Gonna do that.

Question: Would you rather have patches for the branch develop instead of master?